README.md

```diff
@@ -142,21 +141,21 @@ example: `/confluence unsubscribe "Project A Subscription"`.
 **Maintainer:** [@jfrerich](https://github.com/jfrerich)
```

Comment:
Great question, @catalintomai! Yes - we should, particularly in cases where maintainers listed for a given integration were listed as Mattermost staff members (rather than community-led contributions).